eradiate/scenes/measure/_core.py

```diff
@@ -672,29 +752,49 @@ def _to_dataset_helper_data_values(
             * sensor;
             * pixel index.
         """
-        if not eradiate.mode().has_flags(ModeFlags.MTS_MONO | ModeFlags.ANY_CKD):
-            raise UnsupportedModeError(supported=("monochromatic", "ckd"))
 
-        data = np.full(
-            (
-                len(spectral_coords),
-                len(sensor_ids),
-                *film_size,  # Note: Row-major order (width x comes last)
-            ),
-            np.nan,
-        )
+        if eradiate.mode().has_flags(ModeFlags.ANY_MONO | ModeFlags.ANY_CKD):
+            data = np.full(
+                (
+                    len(spectral_coords),
+                    len(sensor_ids),
+                    *film_size,  # Note: Row-major order (width x comes last)
+                    1,
+                ),
+                np.nan,
+            )
 
-        for i_spectral, spectral_coord in enumerate(spectral_coords):
-            for i_sensor, sensor_id in enumerate(sensor_ids):
-                # Note: This doesn't handle heterogeneous sensor film sizes
-                # (i.e. cases in which sensors have different film sizes).
-                # To add support for it, blitting is probably a good approach
-                # https://stackoverflow.com/questions/28676187/numpy-blit-copy-part-of-an-array-to-another-one-with-a-different-size
-                data[i_spectral, i_sensor] = raw[spectral_coord]["values"][sensor_id][
-                    ..., 0
-                ]
-                # This latter indexing selects only one channel in the raw data
-                # array: this works with mono variants but will fail otherwise
+            for i_spectral, spectral_coord in enumerate(spectral_coords):
+                for i_sensor, sensor_id in enumerate(sensor_ids):
+                    # Note: This doesn't handle heterogeneous sensor film sizes
+                    # (i.e. cases in which sensors have different film sizes).
+                    # To add support for it, blitting is probably a good approach
+                    # https://stackoverflow.com/questions/28676187/numpy-blit-copy-part-of-an-array-to-another-one-with-a-different-size
+                    data[i_spectral, i_sensor, ..., 0] = raw[spectral_coord]["values"][
+                        sensor_id
+                    ][..., 0]
```

Comment:
```suggestion
                    ][..., 0]
                    # This latter indexing selects only one channel in the raw data
                    # array: this works with mono variants but will fail otherwise

```
Please keep the comment, it's useful to understand what this complicated indexing does.